packages/tools/fetch-tool/src/fluidAnalyzeMessages.ts

```diff
@@ -490,6 +496,32 @@ function processOp(
             case ContainerMessageType.BlobAttach: {
                 break;
             }
+            case ContainerMessageType.ChunkedOp: {
+                const chunk = runtimeMessage.contents as IChunkedOp;
+                if (!chunkMap.has(runtimeMessage.clientId)) {
+                    chunkMap.set(runtimeMessage.clientId, {chunks: new Array<string>(chunk.totalChunks), totalSize:0});
+                }
+                const value = chunkMap.get(runtimeMessage.clientId);
+                assert(value !== undefined, "Chunk should be set in map");
+                const chunks = value[0];
+                const chunkIndex = chunk.chunkId - 1;
+                if (chunks[chunkIndex] !== undefined) {
+                    throw new Error("Chunk already assigned");
+                }
+                chunks[chunkIndex] = chunk.contents;
+                value[1] += msgSize;
+                if (chunk.chunkId === chunk.totalChunks) {
+                    opCount = chunk.totalChunks; // 1 op for each chunk.
+                    runtimeMessage = Object.create(runtimeMessage);
+                    runtimeMessage.contents = chunks.join("");
+                    runtimeMessage.type = chunk.originalType;
+                    type = chunk.originalType;
+                    totalMsgSize = value[1];
+                    chunkMap.delete(runtimeMessage.clientId);
+                } else {
+                    return;
+                }
+            }
             default: {
```

Comment:
So I think the core of the problem (why errors are not easily diagnosable) is that default case.
Ideally default case should use unreachableCase()
Take a look at isRuntimeMessage(), we should group all those values into some type and leverage it to
- Fail if something comes here outside of this type.
- List all cases here one by one
- Use unreachableCase() for anything else - compiler will detect if that type have additions and will force next developer to add it here.

Reply:
Done